### DISCUSSION TITLE
[Share] Integration between Obsidian Zoom and Another Quick Switcher Plugin

### DISCUSSION THREAD

**Discussion by FelipeRearden** [vslinko/obsidian-zoom#84] [Ideas]
Hello Friends 👋 

I have a workflow between two plugins: Obsidian Zoom and Another Quick Switcher (AQS) that I would like to share with the community.

AQS has a feature that let the user navigate between all headers from the Active Note 
⦿ Open the AQS Modal;
⦿ Select a Header or Search using the text field input;
⦿ The Caret Cursor will jump to the Selected header;

This is very useful when you have a note with a lot of headers

💡 Based on this, I had an idea to integrate the workflow described above with Obsidian Zoom. This is my idea ....

https://user-images.githubusercontent.com/85180696/201516597-265ecfd5-8b39-4a84-b0ed-8951e37a8e0c.MOV

I created an Feature Request issue on AQS repository. If you like this idea, please give a 👍 in my post to show to the AQS developer our need for this feature 🙏🙏🙏

→ The Post is here : https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/121#issue-1445687973

Thank you very much for reading this !

Have a great day !!!

**Comment by vslinko** [comment]
Hi @FelipeRearden!

Nice workflow, I like that idea! Added my 👍 :)

**Reply by FelipeRearden** [reply]
> Nice workflow, I like that idea! Added my 👍 :)

Thank you very much for the support @vslinko !!!!!!!!
